rationale.md

```diff
@@ -511,22 +511,22 @@ Both `DelimArray` and `DelimSequence` accept optional overrides for:
 
 ### Binary response handling
 
-The popular request libraries currently used by the RTLs for both Python and the Node-based Typescript SDK treat response content as string by default, with binary mode as an opt-in setting. This results in the download of binary content like images or PDFs from SDK methods to be corrupted.
+The popular request libraries currently used by the RTLs for both Python and the Node-based Typescript SDK treat response content as string by default, with binary mode as an opt-in setting. By default, this results in the download of binary content like images or PDFs from SDK methods to be corrupted.
 
-Until we've [resolved this issue](#binary-response-detection) with replacement HTTP request methods, the code generator will assist with binary responses in the following ways (the provided examples in Typescript).
+We've resolved this issue by setting up the request methods to process requests as binary by default, with the RTL using the `Content-Type` of the response to determine whether it's binary before returning the response payload to the SDK method.
 
 #### Response can be either binary and string
 
 A **mixed mode** response that can be either binary or string (e.g. text or json) has a note placed into the method header.
 
 ```typescript
 /**
-  * POST /queries/run/{result_format} -> string}
-  *
-  * **Note**: Binary content may be returned by this method. Add `binaryMode` to the `options` parameter to correctly receive binary content
-  */
+ * POST /queries/run/{result_format} -> string
+ *
+ * **Note**: Binary content may be returned by this method. Add `binaryMode` to the `options` parameter to correctly receive binary content
```

Comment:
Still relevant?

Reply:
Removed the "Add binaryMode ..." bits. Thanks!